mlos_bench/mlos_bench/tunables/tunable.py

```diff
@@ -115,6 +123,8 @@ def _sanity_check(self) -> None:
                 raise ValueError(f"Values must be None for the numerical type tunable {self}")
             if not self._range or len(self._range) != 2 or self._range[0] >= self._range[1]:
                 raise ValueError(f"Invalid range for tunable {self}: {self._range}")
+            if self._quantization is not None and self._quantization <= 1:
+                raise ValueError(f"Number of quantization points is <= 1: {self}")
```

Comment:
I think I only enforce `minimum: 1` in the schema.  Could adjust that up to 2.

Reply:
But, not sure how that works for a float param with range [0, 4.5] and with quantization factor 1.5 for instance.

Reply:
For that matter, this is also valid:

```python
import ConfigurationSpace as CS
cs = CS.ConfigurationSpace()
cs.add_hyperparameter(CS.Float(name="float", bounds=(0, 1), q=0.1))
cs.sample_configuration()
```
```python
cs.sample_configuration()
Configuration(values={
  'float': 0.30000000000000004,
})
```

Reply:
that code runs, but `mypy` will issue a warning, because `CS.Float` type signature does *not* allow `float q`

Reply:
I submitted an [Issue](https://github.com/automl/ConfigSpace/issues/349) and [PR](https://github.com/automl/ConfigSpace/pull/350) to fix that upstream.
Meantime, I'd say we just add `type: ignore[arg]` or whatever to ignore that part since it's definitely supported behind the scenes by ConfigSpace - just a type hints bug.

Reply:
Weird that this didn't fire for me though.  I wonder what's different about mypy in Windows.

Reply:
agree. mypy does not complain on my Windows PC, so I think we can merge it